### DISCUSSION TITLE
Reduce xml size by auto-closing empty tags

### DISCUSSION THREAD

**Discussion by NicoAdrian** [manoelcampos/xml2lua#72] [Ideas]
Is there any way to reduce the size of the xml string generated via `xml2lua.toXml` ? I'm talking about using auto closing tags when there is no text, no unnecessary `\n` etc...

**Comment by manoelcampos** [comment]
Hello @NicoAdrian, 
Currently, there is no way to do that.
Unfortunately, this is not a feature I'm planning to do, but Pull Requests are welcome.
